sources/packages/backend/apps/api/src/route-controllers/application/application.controller.service.ts

```diff
@@ -275,8 +275,9 @@ export class ApplicationControllerService {
    */
   processApplicationSupportingUserDetails(
```

Comment:
I am not sure if this method's return type needs to be a DTO because  the output of this method is merged with other information at controller and it returns an Out DTO. 

![image](https://user-images.githubusercontent.com/54600590/211059554-9a21ed4c-6ed9-4efc-84a4-11e8080c6b45.png)

Reply:
Yes, I got what you're saying but my first understanding was that if that is part of DTO, then it should also be a DTO... what do you guys say? @ann-aot, @andrewsignori-aot @guru-aot

Reply:
If the object is "leaving" the API it will be a DTO, if the object is part of a process that will return information to the controller and the controller will then use this information to generate the DTO, it does not need to be named as DTO.
For this case, it seems that the controller is just retuning the received DTO that is part of a bigger DTO composed of many other DTOs. Since we have the "big" DTO we can keep the "small" objects without the DTO name, IMO.

![image](https://user-images.githubusercontent.com/61259237/211096764-a59815e2-80f9-4b4d-8771-1655faa7b3c1.png)

That said, these DTOs seem to be in a `system.dto.ts` and they need be moved also.

Reply:
ok. Moved dtos to `application.dto.ts`